src/crypto.cpp

```diff
@@ -1475,7 +1477,22 @@ OcspResponse::getCertificateStatus() const
 {
     int ret;
     unsigned int status;
+    // MSVC/vcpkg GnuTLS cast workaround
+#if _WIN32 && (GNUTLS_VERSION_NUMBER >= 0x030807)
+    ret = gnutls_ocsp_resp_get_single(response, 
+                                      0, 
+                                      NULL, 
+                                      NULL, 
+                                      NULL, 
+                                      NULL, 
+                                      reinterpret_cast<gnutls_ocsp_cert_status_t*>(& status), 
+                                      NULL, 
+                                      NULL, 
+                                      NULL, 
+                                      NULL);
+#else
     ret = gnutls_ocsp_resp_get_single(response, 0, NULL, NULL, NULL, NULL, &status, NULL, NULL, NULL, NULL);
+#endif
```

Comment:
please move this change to a separate PR

Reply:
Done in #755